op_crates/fetch/26_fetch.js

```diff
@@ -1027,22 +1070,19 @@
         init &&
         "credentials" in init &&
         init.credentials &&
-        ["omit", "same-origin", "include"].indexOf(init.credentials) !== -1
+        ["omit", "same-origin", "include"].includes(init.credentials)
       ) {
         this.credentials = init.credentials;
       }
     }
 
     clone() {
       if (this.bodyUsed) {
-        throw TypeError(BodyUsedError);
+        throw new TypeError(BodyUsedError);
       }
 
       const iterators = this.headers.entries();
-      const headersList = [];
-      for (const header of iterators) {
-        headersList.push(header);
-      }
+      const headersList = [...iterators];
```

Comment:
I just passed through, making minor tweaks, but I'm not 100% sure if using the iterable like this was safe.

Reply:
Reverted these small changes.